app/socket-apis/get-points-of-ids.js

```diff
@@ -0,0 +1,44 @@
+// https://github.com/EnCiv/civil-pursuit/issues/203
+const Points = require('../models/points')
+const { ObjectId } = require('mongodb')
+
+async function getPointsOfIds(ids, callback, currentUserId) {
+  try {
+    // Fetch points by _id
+    const points = await Points.aggregate([{ $match: { _id: { $in: ids.map(id => new ObjectId(id)) } } }]).toArray()
+
+    // Fetch why points where parentId is in the ids
+    const whypoints = await Points.aggregate([
+      { $match: { parentId: { $in: ids.map(id => new ObjectId(id)) } } },
+    ]).toArray()
+
+    // Reduce points into an object
```

Comment:
I admit the issue description made it look like this api should return objects, as done here, but this api should just return arrays, and it was the browser side code that should convert it to object.  But that's my bad, I will clean up the issue description to clarify this, but please just return points and whyPoints, not the byId tables.